### ISSUE TITLE
Link is not extracted at v0.3.3

### ISSUE THREAD

**Issue by dalance** [google/mdbook-i18n-helpers#202]
I found an unexpected behaviour change.
At v0.3.3, link doesn't seem to be extracted.
The reproduce proredure is below:

* generate messages.pot at https://github.com/rust-lang/rust-by-example.
* get the following msgid at v0.3.2

```
#: src/index.md:3
msgid ""
"[Rust](https://www.rust-lang.org/) is a modern systems programming language "
"focusing on safety, speed, and concurrency. It accomplishes these goals by "
"being memory safe without using  garbage collection."
msgstr ""
```

* get the following msgid at v0.3.3

```
#: src/index.md:3
msgid ""
"Rust is a modern systems programming language focusing on safety, speed, and "
"concurrency. It accomplishes these goals by being memory safe without using "
"garbage collection."
msgstr ""
```

**Comment by dalance** [comment]
@mgeisler @kdarkhan 
Could you check this issue and #203?
This seems to be critical because generated messages.pot becomes broken.

**Comment by kdarkhan** [comment]
Thanks for the fix @dalance, sorry missed the notification.

I merged and will try to release a new version once I get https://github.com/google/mdbook-i18n-helpers/pull/204 approved.

**Comment by kdarkhan** [comment]
Just published version 0.3.4 to crates.
